manifest-build-tools/pushToBintray.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+#Sample Usage: pushToBintray.sh username api_key bintray_repo bintray_package_name bintray_package_version local_package_path
+
+set -e
+set -x
+
+BASEDIR=$(dirname "$0")
+
+BINTRAY_CONFIG_FILE=$BASEDIR/bintray.conf
+
+if [ ! -f "$BINTRAY_CONFIG_FILE" ]; then
+    echo "The bintray.conf is missing. Exiting now..."
+    exit 1
+fi
+
+source $BINTRAY_CONFIG_FILE
+BINTRAY_USER=$1
+BINTRAY_API_KEY=$2
+BINTRAY_REPO=$3
+BINTRAY_PCK=$4
+BINTRAY_PCK_VERSION=$5
+PACKAGE_PATH=$6
+
+main() {
+  CURL="curl -u${BINTRAY_USER}:${BINTRAY_API_KEY} -H Content-Type:application/json -H Accept:application/json"
+  if (check_package_exists); then
```

Comment:
1. if package exist, then check_package_exists() returns 1, then goes into the IF block, but it echo "does not exist" ????

2. I think it can be an option for script user to over-ride an existing package with same name.